Langs/0407.lang

```diff
@@ -78,22 +79,22 @@ Done=Fertig
 Loading WMIC=Lade WMIC
 Warning=Warnung
 Your Computer is NOT ready for Windows 11, you can join the Discord using the Discord Icon if you need assistance.=Ihr Computer ist NICHT für Windows 11 bereit. Sie können dem Discord über das Discord-Symbol beitreten, wenn Sie Hilfe benötigen.
-Your Computer is ready for Windows 11. You should receive the option to upgrade between October 5th 2021 and Fall 2022.=Ihr Computer ist bereit für Windows 11. Sie sollten zwischen dem 5. Oktober 2021 und Herbst 2022 die Möglichkeit zum Upgrade erhalten.
+Language Switcher currently disabled with Group Policy=Sprachumschaltung derzeit mit Gruppenrichtlinie deaktiviert.
+Theme Switcher currently disabled with Group Policy.=Themenumschaltung ist derzeit durch Gruppenrichtlinien deaktiviert
+Default - No Theme Files Found=Standard - Keine Designdateien gefunden
 Guides=Anleitungen (englisch)
 Windows 11 Requirements=Win 11 Vorraussetzungen
 Convert Disk to GPT=GPT-Konvertierung
 Enable TPM=TPM-Aktivierung
 Enable Secure Boot=Secure Boot-Aktiverung
 Skip CPU && TPM=CPU && TPM-Prüfung überspringen
 Get Windows 11 Now=Anleitung zum Upgrade
-May Report DirectX 12 Incorrectly=Kann DirectX 12 falsch melden
```

Comment:
Why was this line removed? can still see in current file
https://github.com/rcmaehl/WhyNotWin11/blob/main/Langs/0000.lang#L60

Reply:
nvm im blind